src/poggit/ci/builder/ProjectBuilder.php

```diff
@@ -570,7 +570,8 @@ protected function lintManifest(RepoZipball $zipball, BuildResult $result, strin
             $status->className = $manifest["main"];
             $result->addStatus($status);
         }
-        if(!$zipball->isFile($mainClassFile = $this->project->path . "src/" . str_replace("\\", "/", $mainClass = $manifest["main"]) . ".php")) {
+        $mainClassFile = $this->project->path . "src/" . str_replace((trim($manifest["src-namespace-prefix"]??"") === "" ?: str_replace("\\", "/", $manifest["src-namespace-prefix"])."/"), "", str_replace("\\", "/", $mainClass = $manifest["main"])) . ".php";
```

Comment:
Could you split down this line please? Poggit doesn't really have good code style, but we don't have to torture ourselves further with a 250-character-long line.

Reply:
Better?